tests/test_client.py

```diff
@@ -77,7 +84,21 @@ def test_login_user(self):
 
         response = guest.login_user(username = "openreview.net", password = "1234")
         assert response
-        print(response)
+
+        response = guest.login_user(username = "openreview.net", password = "1234", expiresIn=4000)
+        assert response
+
+    def test_login_expiration(self):
+        client = openreview.Client(username = "openreview.net", password = "1234", tokenExpiresIn=3)
+        group = client.get_group("openreview.net")
+        assert group
+        assert group.members == ['~Super_User1']
+        time.sleep(4)
+        try:
+            group = client.get_group("openreview.net")
+        except openreview.OpenReviewException as e:
+            error = e.args[0]
+            assert e.args[0]['name'] == 'TokenExpiredError'
```

Comment:
Not sure why, but this test is no longer passing. Seems like the API is returning a different error: `{'type': 'forbidden', 'path': 'id', 'value': 'openreview.net', 'user': 'guest_1620851435577'}`

Reply:
This is happening because I commented this code in the backend, because `TokenExpiredError` was not handled by UI to retrieve a new token. I'll comment this test too